### ISSUE TITLE
Command: Status

### ISSUE THREAD

**Issue by sondreb** [block-core/blockcore-tipbot#10]
Consider adding a "status" command that can return information such as block height, which can be used to verify that the tipbot chain is fully synced. Other status information could be shown too.

Statistics would be interesting too, how many total users, etc.

**Comment by MudDev** [comment]
Ah, yeah! Can be part of that dashboard your dashboard too

**Comment by sondreb** [comment]
Yes, statistics is probably just better for the dashboard, but a status might be useful to quickly verify that the bot is fully operational.

**Comment by MithrilMan** [comment]
When I'll push my bot you can take some part of it (or even merge), I've some statistic module already implemented, with charts

**Comment by sondreb** [comment]
Internally the tipbot itself should have this information available, so that it can avoid performing certain operations unless the node it is connected to, has connections.
